include/boost/statechart/detail/rtti_policy.hpp

```diff
@@ -11,11 +11,13 @@
 #include <boost/assert.hpp>
 #include <boost/config.hpp> // BOOST_MSVC
 #include <boost/detail/workaround.hpp>
+#if !defined(BOOST_STATECHART_USE_NATIVE_RTTI)
```

Comment:
Do we need to worry about `BOOST_NO_RTTI` here?

Is this macro new? - does it need documentation and additional test cases?

Reply:
This is to avoid including Boost.TypeIndex when not needed. The macro is not new, it is already documented and is tested by *Native tests.